AlwaysOnKotlin/Wearable/src/androidTest/java/com/example/android/wearable/wear/alwayson/MainActivityTests.kt

```diff
@@ -99,10 +100,12 @@ class MainActivityTests {
             )
         onView(withId(R.id.state)).check(matches(withText(context.getString(R.string.mode_active_label))))
         onView(withId(R.id.draw_count)).check(matches(withText(context.getString(R.string.draw_count_label, 1))))
+
+        scenario.moveToState(Lifecycle.State.STARTED)
```

Comment:
If we don't pause the activity, then `runTest` fails with uncompleted work. This makes sense: we are always enqueuing an infinite amount of work, so we need to cancel it before finishing `runTest` by pausing the activity.

Reply:
I'm confused, should there be some call to pause it? I'm new to this API though, so I might be missing something. (If so, ignore this.) :)

Also, should we use your comment as a comment in the code?

Reply:
`moveToState` `STARTED` will move from `RESUMED` to `STARTED`, thereby pausing it. I'll add my comment inline to make that clear!

Reply:
Updated!

Reply:
Perfect, thanks!